### ISSUE TITLE
No music via BT loudspeaker

### ISSUE THREAD

**Issue by titus483** [DeaDBeeF-Player/deadbeef#2757]
### Steps to reproduce the problem
I am able to connect my laptop via BT to a BOSE speaker. But when I want to play music the BT speaker remains silent. I can only hear the music playing on the laptop speaker (which is not really great).
### What's going on? Describe the problem in as much detail as possible.
I had the same problem at the beginning with Clementine. After I installed some packages for gstreamer I was able to listen to the music via the BT speaker. So I know that 
a) the BT loudspeaker works
b) the BT connection is ok.
But why is the sound not "sent" to the BT speaker in DeaDbeef?
### Information about the software:
I assume some piece of software is missing. But which one?
Deadbeef version: 1.8.4
OS: MX-Linux 21

**Comment by Alexey-Yakovenko** [comment]
It sounds more like a setup issue. Did you try finding some new audio device in the list of sound devices in deadbeef preferences? Typically, this should either work automatically and handled by pulseaudio, or you need to select the BT speaker audio device directly in the preferences.

**Comment by titus483** [comment]
Hi,

in my preferences I have on the tab Sound Output:

Output Extension: PulseAudio output plugin (selectable: ALSA output plugin, Null output plugin, OSS output plugin)
Output Device:  Standard Output Device (greyed out, hence no other selection possible)

Then I hear the music via the laptop's internal speaker. I played a bit with the output extensions, but the I don't hear anything.

P.S. I have a German language version installed. This is a copy of a PM.

**Comment by Alexey-Yakovenko** [comment]
Yeah, pulseaudio output in deadbeef will only play through the default device, as configured in pulseaudio itself.
Perhaps you will have a better luck with this alternative pulseaudio plugin? It's supposed to be more feature-rich.
http://sourceforge.net/projects/deadbeef/files/plugins/x86_64/ddb_output_pulse2-v1.2-linux-x86_64.zip/download

**Comment by titus483** [comment]
That's great. After installation of the plugin I was able to select the BT speaker and: it works. 
Thanks a lot!

**Comment by Alexey-Yakovenko** [comment]
Great!
